packages/react-components/react-nav-preview/src/index.ts

```diff
@@ -2,13 +2,18 @@ export { Nav, navClassNames, renderNav_unstable, useNav_unstable } from './Nav';
 export type { NavProps, NavSlots, NavState } from './Nav';
 // these should probably be explicity as they are above.
 export {
-  NavGroup,
-  navGroupClassNames,
-  renderNavGroup_unstable,
-  useNavGroup_unstable,
-  useNavGroupStyles_unstable,
-} from './components/NavGroup/index';
-export type { NavGroupProps, NavGroupSlots, NavGroupValue, NavGroupState } from './components/NavGroup/index';
+  NavCategoryItem as NavGroup,
+  navCategoryItemClassNames as navGroupClassNames,
+  renderNavCategoryItem_unstable as renderNavGroup_unstable,
+  useNavCategoryItem_unstable as useNavGroup_unstable,
+  useNavCategoryItemStyles_unstable as useNavGroupStyles_unstable,
+} from './components/NavCategoryItem/index';
+export type {
+  NavCategoryItemProps as NavGroupProps,
+  NavCategoryItemSlots as NavGroupSlots,
+  NavCategoryItemValue as NavGroupValue,
+  NavCategoryItemState as NavGroupState,
+} from './components/NavCategoryItem/index';
```

Comment:
What's the plan with these exports? Since this is still a preview package, breaking changes are allowed. Do you plan to remove the NavGroup exports and replace them with NavCategoryItem eventually?

Reply:
Yeah - I'll go ahead and update them now.